### ISSUE TITLE
Fishnet 1

### ISSUE THREAD

**Issue by samuelstevens** [samuelstevens/swin-transformer#5]
Dataset: Fishnet
Data: 100%
Model: Best iNat21 

Use a pre-trained swin-v2-base model pre-trained on iNat21 with the best validation loss, fine-tune it on Fishnet, then report the accuracy (and preferably attach training logs, link to WandB dashboards, etc).
